loader/paths.go

```diff
@@ -105,7 +105,7 @@ func ResolveServiceRelativePaths(workingDir string, s *types.ServiceConfig) {
 	}
 }
 
-func absPath(workingDir string, filePath string) string {
+func AbsPath(workingDir string, filePath string) string {
```

Comment:
Do we need to export this? This func is slightly problematic TBH, so I'd rather not expand its usage (I actually got rid of it/replaced it in #436)

Reply:
AFAICT the `RealAbsPath` you introduce is a drop-in replacement for this one, isn't it ? Then could just use it